tutorials/spa-dox-create-automation/spa-dox-create-automation.md

```diff
@@ -140,17 +136,19 @@ primary_tag: software-product>sap-process-automation
 
 [ACCORDION-BEGIN [Step 5: ](Annotate and Activate the Document Template)]
 
-    You are now navigated to Document Information Extractor. It allows you to edit and annotate the template.
+1. You are now navigated to Document Information Extractor. It allows you to edit and annotate the template. To work on Document Information Extractor, please check if you have been assigned to `Document_Information_Extraction_UI_Templates_Admin` role in SAPBTP Cockpit. Otherwise, add the role manually. You would error as below if you do not have DOX roles.
```

Comment:
"You would error as below if you do not have DOX roles."

I think that there is a verb missing.